pkg/controller/volume/attachdetach/util/util.go

```diff
@@ -50,10 +50,7 @@ func CreateVolumeSpec(podVolume v1.Volume, pod *v1.Pod, nodeName types.NodeName,
 		claimName = ephemeral.VolumeClaimName(pod, &podVolume)
 	}
 	if claimName != "" {
-		klog.V(10).Infof(
-			"Found PVC, ClaimName: %q/%q",
-			pod.Namespace,
-			claimName)
+		logger.V(10).Info("Found PVC", "claim", klog.KRef(pod.Namespace, claimName))
```

Comment:
Let's use "PVC" as key for a persistent volume claim, it's less ambiguous and more consistent ("PVC" is already used). Applies also elsewhere.

Reply:
done